pkg/vetter/invalidserviceforjwtpolicy/vet.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2019 Aspen Mesh Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package invalidserviceforjwtpolicy
+
+import (
+	"strings"
+
+	"github.com/aspenmesh/istio-client-go/pkg/apis/authentication/v1alpha1"
+	authv1alpha1 "github.com/aspenmesh/istio-client-go/pkg/client/listers/authentication/v1alpha1"
+	"github.com/golang/glog"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	listersvV1 "k8s.io/client-go/listers/core/v1"
+
+	apiv1 "github.com/aspenmesh/istio-vet/api/v1"
+	"github.com/aspenmesh/istio-vet/pkg/vetter"
+)
+
+const (
+	vetterID                  = "InvalidServiceForJWTPolicy"
+	invalidServiceNoteType    = "invalid-service-for-jwt-authentication-policy"
+	invalidServiceNoteSummary = "Target services must have valid service port names"
```

Comment:
Since the vetter ID has `forJWTPolicy’ in it, would it be better to exclude it from the note type so that it can be shorter?

invalidServiceNoteType -> invalidTargetServicePortName = “invalid-target-service-port-name”